todo_app/app.py

```diff
@@ -1,15 +1,24 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items, add_item
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/index')
 def index():
-    return 'Hello World!'
+	todolists = get_items()
+	return render_template('index.html', todolists = todolists)
+
 
+@app.route('/addtodoitem', methods=['POST'])
+def addtodoitem():
+	add_item(request.form['item_title'])
+	todolists = get_items()
+	return render_template('index.html', todolists = todolists)
```

Comment:
You should `return redirect(url_for('index'))` so that:
- You don't duplicate code
- The browser will show the correct URL
- The user can refresh the page without resubmitting the form